src/app/routes/applications/new-application/new-application.component.html

```diff
@@ -82,7 +82,10 @@ <h4 mat-dialog-title class="mb-0 d-flex justify-content-between align-items-cent
                   *ngIf="appForm.get('appName').value"
                   class="ml-3 card card-result d-flex flex-row justify-content-start align-items-center shadow-none d-flex pt-3 pb-2 mb-3 mr-3">
                   <span class="pb-2">
-                    {{ appForm?.value?.appName + '.' + ENSPrefixes.Application + '.' + appForm?.value?.orgNamespace }}
+                    {{
```

Comment:
What about 
```suggestion
                    {{ namespace }}
```
Where `namespace` would be updated on service layer?